_template_python/delphi_NAME/run.py

```diff
@@ -2,20 +2,34 @@
 """Functions to call when running the function.
 
 This module should contain a function called `run_module`, that is executed
-when the module is run with `python -m MODULE_NAME`.
+when the module is run with `python -m MODULE_NAME`.  Its lone argument should be a nested
```

Comment:
The "lone argument" here is the argument to run_module() and not to the command line command right? 

```suggestion
when the module is run with `python -m MODULE_NAME`.  `run_module`'s lone argument should be a nested
```